.circleci/scripts/binary_linux_test.sh

```diff
@@ -95,7 +95,13 @@ if [[ "$PACKAGE_TYPE" == conda ]]; then
     conda install \${EXTRA_CONDA_FLAGS} -y "\$pkg" --offline
   )
 elif [[ "$PACKAGE_TYPE" != libtorch ]]; then
-  pip install "\$pkg" --index-url "https://download.pytorch.org/whl/nightly/${DESIRED_CUDA}"
+  if [[ "$(uname -m)" == aarch64 ]]; then
+    # Using "extra-index-url" until all needed aarch64 dependencies are
+    # added to "https://download.pytorch.org/whl/nightly/"
+    pip install "\$pkg" --extra-index-url "https://download.pytorch.org/whl/nightly/${DESIRED_CUDA}"
+  else
```

Comment:
Can you list all the dependencies here, so that they could be uploaded.

Reply:
@atalman, I attempted a run without --extra-index-url and am unable to get the needed dependencies. Work to get a list of the dependencies from the exact binary. Is the aarch64 dependent packages at https://download.pytorch.org/whl/nightly/cpu, or is there another url for it?